exercises/hello-world/hello_test.go

```diff
@@ -2,7 +2,7 @@ package hello
 
 import "testing"
 
-// Define a function HelloWorld(string) string.
+// Define a function World(string) string.
```

Comment:
The name `World` is a bit awkward, as it would read: `World("Alice")` or `World("")`. I think `Hello` would make more sense in this context. E.g. `Hello("Alice")` or `Hello("")`.